scripts/images/convert_qcow2_ova.py

```diff
@@ -420,6 +433,10 @@ def convert_qcow2_ova(imageUrl, imageSize, imageName, imageDist, rhnUser, rhnPas
             print('ERROR: Resizing failed')
             sys.exit(2)
 
+        if imageDist == 'rhel' or imageDist == 'centos':
+            print("Preparing rhel image...")
```

Comment:
print("Preparing ", imageDist, " image...") ??

Reply:
It was just a copy-paste, but I will take care.